plugin_upload.py

```diff
@@ -4,16 +4,16 @@
         Authors: A. Pasotti, V. Picavet
         git sha              : $TemplateVCSFormat
 """
-from __future__ import print_function
-
+from builtins import input
 from future import standard_library
+from optparse import OptionParser
 
-standard_library.install_aliases()
-from builtins import input
-import sys
 import getpass
+import sys
 import xmlrpc.client
-from optparse import OptionParser
+
+
+standard_library.install_aliases()
```

Comment:
Some leftover `future`? `future` without the dunders?
Not completely sure what it is for, but reading https://python-future.org/_modules/future/standard_library.html seems like it can be removed.

Reply:
Done.